public/javascript/main.js

```diff
@@ -377,4 +376,81 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
-});
+
+// Google Analytics event tracking
+$('[data-ga-track-select]').on('focus',function(event){
+    ga('send', 'event', 'focus', $(this).attr('data-ga-tack-select'), document.title);
+})
+
+$('[data-ga-track-click]').on('click', function(event){
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
Yes, I should remove that. It's left over from my attempts at trying to prevent the dropdown list from firing.